[31m[1mERROR   [0m personas.models:models.py:131 Error al enviar el correo: database table is locked: titulos